db/migrate/20200617014509_add_subscriber_email_index_to_user_subscriptions.rb

```diff
@@ -0,0 +1,45 @@
+class AddSubscriberEmailIndexToUserSubscriptions < ActiveRecord::Migration[6.0]
+  disable_ddl_transaction!
+
+  def up
+    if index_exists?(:user_subscriptions, %i[subscriber_id user_subscription_sourceable_id user_subscription_sourceable_type])
+      remove_index :user_subscriptions,
+                   column: %i[subscriber_id user_subscription_sourceable_id user_subscription_sourceable_type],
+                   algorithm: :concurrently
+    end
+
+    if !index_exists?(:user_subscriptions, :subscriber_email)
```

Comment:
I'm surprised Rubocop didn't try to change this to `unless index_exists?`

Reply:
I prefer `unless index_exists?` as well. It seems we have used `!index_exists?` as well 😅 . It's a dead tie with this included. I switched it 💯